clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -342,7 +344,9 @@ protected synchronized void bootstrap(Optional<Set<String>> managedClients) {
     // Subscribe all bootstrap version partitions.
     storeNameToBootstrapVersionMap.forEach((storeName, version) -> {
       List<Integer> partitions = storeNameToPartitionListMap.get(storeName);
+      String versionTopic = version.kafkaTopicName();
       LOGGER.info("Bootstrapping partitions " + partitions + " for " + version.kafkaTopicName());
```

Comment:
nit: might just use the `versionTopic` in this logger statement. Besides, we can use log interpolation!
```suggestion
      LOGGER.info("Bootstrapping partitions {} for {}", partitions, versionTopic);
```

Reply:
Thank you for spotting this - I believe we (it is probably me fixing these batch of files) missed this one : )